packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -5182,9 +5182,6 @@ export default class Meeting extends StatelessWebexPlugin {
     if (LAYOUT_TYPES.includes(layoutType)) {
       layoutInfo.layoutType = layoutType;
     }
-    else {
```

Comment:
I think we should still fail if a non-undefined layoutType is passed in that doesn't match any valid value. We should maybe also update the docs for this function to say that layoutType is optional?

Reply:
I've updated the source to address this. layoutType is now optional in JSDoc. This conditional needs to stay due to a previously existing layoutInfo validation check later within this function.

Reply:
ok, but now we silently ignore any invalid `layoutType` values?

Reply:
I believe I understand now. I've made some changes, please re-review.